enterprise/api_client/enterprise.py

```diff
@@ -0,0 +1,50 @@
+
+# -*- coding: utf-8 -*-
+"""
+Client for communicating with the Enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+from django.core.cache import cache
+
+from enterprise.api_client.lms import LmsApiClient
+from enterprise.utils import get_cache_key, traverse_pagination
+
+
+class EnterpriseApiClient(LmsApiClient):
```

Comment:
this makes me wonder why we don't have the lms and ecommerce pull from a client hosted in the enterprise package instead of having their own version of the client in each place. maybe that's something we can clean up later...